diff -urN traffic-current/src/routines.c traffic-current.new/src/routines.c
--- traffic-current/src/routines.c	2003-12-14 00:38:57.000000000 +0100
+++ traffic-current.new/src/routines.c	2003-12-29 20:56:00.000000000 +0100
@@ -28,27 +28,27 @@
 } 
 
 void usage() {
-	printf("Available options:
-\t-f <filename>
-\t-i <interface>
-\t-u <unit>
-\t-U <unit>
-\t-r <refresh>
-\t-l <logfile>
-\t-c <class> (xxx.xxx.xxx.xxx/xx)
-\t-s <column>
-\t-R (reverse sorting)
-\t-t (enable summarize line)
-\t-C <columns list>
-\t-e <filename>
-\t-E <host or network>
-\t-x <filename>
-\t-X <host or network>
-\t-d (daemon mode)
-\t-D (discovery mode)
-\t-P (private mode)
-\t-T (timeout)
-\t-v (print version)\n");
+	printf("Available options:\n"
+"\t-f <filename>\n"
+"\t-i <interface>\n"
+"\t-u <unit>\n"
+"\t-U <unit>\n"
+"\t-r <refresh>\n"
+"\t-l <logfile>\n"
+"\t-c <class> (xxx.xxx.xxx.xxx/xx)\n"
+"\t-s <column>\n"
+"\t-R (reverse sorting)\n"
+"\t-t (enable summarize line)\n"
+"\t-C <columns list>\n"
+"\t-e <filename>\n"
+"\t-E <host or network>\n"
+"\t-x <filename>\n"
+"\t-X <host or network>\n"
+"\t-d (daemon mode)\n"
+"\t-D (discovery mode)\n"
+"\t-P (private mode)\n"
+"\t-T (timeout)\n"
+"\t-v (print version)\n");
 	exit(0);
 }
 
diff -urN traffic-current/src/show_stats.c traffic-current.new/src/show_stats.c
--- traffic-current/src/show_stats.c	2003-11-22 23:12:59.000000000 +0100
+++ traffic-current.new/src/show_stats.c	2003-12-29 20:55:17.000000000 +0100
@@ -508,45 +508,45 @@
 {
 	extern WINDOW	*win;
 
-	wprintw(win, "Keys:
-	a + <column>    - add column
-	d + <column>    - remove column
-	u + <unit>      - change data unit size
-	U + <unit>      - change total data unit size
-	s + <column>    - sort
-	t               - total line
-	r               - reset statistics
-	?   - help
-	q   - quit
-
-Columns:
-	h   - host ip
-	c   - current data transfer
-	a   - average data transfer
-	m   - maximum data transfer
-	s   - total data transfer
-	p   - current packets flow
-	o   - average packets flow
-	m   - maximum packets flow
-	P   - total packets flow
-	M   - MAC address
-	H   - hostname
-	t   - TCP data flow
-	u   - UDP data flow
-	i   - ICMP date flow
-	T   - Total TCP data flow
-	U   - Total UDP data flow
-	I   - Total ICMP date flow
-
-Units:
-	b - Bits
-	B - Bytes
-	k - Kilobits
-	K - Kilobytes
-	m - Megabits
-	M - Megabytes
-	g - Gigabits
-	G - Gigabytes\n");
+	wprintw(win, "Keys:\n"
+"	a + <column>    - add column\n"
+"	d + <column>    - remove column\n"
+"	u + <unit>      - change data unit size\n"
+"	U + <unit>      - change total data unit size\n"
+"	s + <column>    - sort\n"
+"	t               - total line\n"
+"	r               - reset statistics\n"
+"	?   - help\n"
+"	q   - quit\n"
+"\n"
+"Columns:\n"
+"	h   - host ip\n"
+"	c   - current data transfer\n"
+"	a   - average data transfer\n"
+"	m   - maximum data transfer\n"
+"	s   - total data transfer\n"
+"	p   - current packets flow\n"
+"	o   - average packets flow\n"
+"	m   - maximum packets flow\n"
+"	P   - total packets flow\n"
+"	M   - MAC address\n"
+"	H   - hostname\n"
+"	t   - TCP data flow\n"
+"	u   - UDP data flow\n"
+"	i   - ICMP date flow\n"
+"	T   - Total TCP data flow\n"
+"	U   - Total UDP data flow\n"
+"	I   - Total ICMP date flow\n"
+"\n"
+"Units:\n"
+"	b - Bits\n"
+"	B - Bytes\n"
+"	k - Kilobits\n"
+"	K - Kilobytes\n"
+"	m - Megabits\n"
+"	M - Megabytes\n"
+"	g - Gigabits\n"
+"	G - Gigabytes\n");
 }
 
 /* Get column width */
